### ISSUE TITLE
error running in electron

### ISSUE THREAD

**Issue by dominictarr** [evbogue/minbay#8]
I tried running this in electron, and it worked the first time, but the second time it did not.
I guessed it was that it set the remote wrong, and I checked `localStorage.remote` was equal to `Not Found`

looks like it was setting the remote to the http response without checking it was really a valid multiserver address.

**Comment by evbogue** [comment]
@dominictarr Fixed in 30decc7b40f2e50af8f1267b047e55fb38f58f8c

Is there library to detect valid multiserver addresses? I did a quick look around and couldn't find anything.

**Comment by dominictarr** [comment]
yeah `ssb-ref` can do it (it has isType for all the important ssb strings)
